quantum/process_keycode/process_digitizer.c

```diff
@@ -0,0 +1,21 @@
+#include "digitizer.h"
```

Comment:
Not sure this should be in `process_keycode/`, tbh. But perhaps it's the folder name that's wrong...

Reply:
Yeah, I think I got mistaken by taking example on the midi implementation that has the task function in the process_keycode folder, but other features have them in the implementation file in the parent directory. I'll fix that for the digitizer and I should make the joystick implementation more consistent with the rest in a separate PR.